service/geopmdpy_test/TestAccess.py

```diff
@@ -142,14 +142,12 @@ def test_write_default_signals(self):
         """
         return_value = (self._signals_expect,
                         self._controls_expect)
-        self._geopm_proxy.PlatformSetGroupAccess = mock.Mock()
-        self._geopm_proxy.PlatformGetGroupAccess = mock.Mock(return_value=return_value)
+        self._geopm_proxy.PlatformSetGroupAccessSignals = mock.Mock()
         self._geopm_proxy.PlatformGetAllAccess = mock.Mock(return_value=return_value)
         with mock.patch('sys.stdin.readlines', return_value=self._signals_expect):
             self._access.run(True, False, False, None, False, False, False, False, False)
-            self._geopm_proxy.PlatformGetGroupAccess.assert_called_once_with('')
-            self._geopm_proxy.PlatformSetGroupAccess.assert_called_once_with('',
-                self._signals_expect, self._controls_expect)
+            self._geopm_proxy.PlatformSetGroupAccessSignals.assert_called_once_with('',
+                self._signals_expect)
```

Comment:
Overall, I see all the calls to Platform[Get|Set]GroupAccess are being removed in favor of testing the new APIs. However, if we are still going to keep the original/existing APIs for now, I would recommend we still continue to test them.

Reply:
Seems like a good idea :) thanks